src-vue/src/components/LocalModCard.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <el-card shadow="hover">
+        <el-switch style="--el-switch-on-color: #13ce66; --el-switch-off-color: #8957e5" v-model="mod.enabled"
+                    :before-change="() => updateWhichModsEnabled(mod)" :loading="global_load_indicator" />
+        <el-popconfirm
+            :title="$t('mods.local.delete_confirm')"
+            :confirm-button-text="$t('generic.yes')"
+            :cancel-button-text="$t('generic.no')"
+            @confirm="deleteMod(mod)"
+        >
+            <template #reference>
+                <el-button type="danger">
+                    {{ $t('mods.local.delete') }}
+                </el-button>
+            </template>
+        </el-popconfirm>
+        {{ mod.name }}
+        <span v-if="mod.version != null">(v{{ mod.version }})</span>
+        <img
+            v-if="mod.thunderstore_mod_string != null"
+            :title="$t('mods.local.part_of_ts_mod') + '\n' + mod.thunderstore_mod_string"
+            src="/src/assets/thunderstore-icon.png"
+            class="image"
+            height="16"
+        />
+    </el-card>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import { invoke } from "@tauri-apps/api";
+import { NorthstarMod } from "../../../src-tauri/bindings/NorthstarMod";
+import { GameInstall } from "../utils/GameInstall";
+import { showErrorNotification, showNotification } from "../utils/ui";
+
+export default defineComponent({
+    name: "LocalModCard",
+    props: {
+        mod: {
+            required: true,
+            type: Object as () => NorthstarMod
+        }
+    },
+    data() {
+        return {
+            global_load_indicator: false,
+        };
+    },
+    computed: {
```

Comment:
If this property doesn't contain anything, you should remove it.

Reply:
Good catch. Removed in c2ae00e5d6c12aca2b637a038416775bc29ae79b ^^